### PR TITLE
[BEAM-8575] Removed MAX_TIMESTAMP from testing data

### PR DESCRIPTION
**Pull request by bumblebee-coming**
Removed MAX_TIMESTAMP from testing data, because the semantics of these extremum elements are not decided yet.

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/#make-reviewers-job-easier).

Post-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

Lang | SDK | Apex | Dataflow | Flink | Gearpump | Samza | Spark
--- | --- | --- | --- | --- | --- | --- | ---
Go | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go/lastCompletedBuild/) | --- | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go_VR_Flink/lastCompletedBuild/) | --- | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go_VR_Spark/lastCompletedBuild/)
Java | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Apex/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Dataflow/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Flink/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Flink_Batch/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Flink_Streaming/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Gearpump/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Samza/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Spark/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Spark_Batch/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_SparkStructuredStreaming/lastCompletedBuild/)
Python | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python2/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Python35/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Python36/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Python37/lastCompletedBuild/) | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_VR_Dataflow/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Py_ValCont/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Python2_PVR_Flink_Cron/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Python35_VR_Flink/lastCompletedBuild/) | --- | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python_VR_Spark/lastCompletedBuild/)
XLang | --- | --- | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_XVR_Flink/lastCompletedBuild/) | --- | --- | ---

Pre-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

--- |Java | Python | Go | Website
--- | --- | --- | --- | ---
Non-portable | [Build Status](https://builds.apache.org/job/beam_PreCommit_Java_Cron/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Python_Cron/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PreCommit_PythonLint_Cron/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Go_Cron/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Website_Cron/lastCompletedBuild/) 
Portable | --- | [Build Status](https://builds.apache.org/job/beam_PreCommit_Portable_Python_Cron/lastCompletedBuild/) | --- | ---

See [.test-infra/jenkins/README](https://github.com/apache/beam/blob/master/.test-infra/jenkins/README.md) for trigger phrase, status and link of all Jenkins jobs.

### CODE REVIEW

File: sdks/python/apache_beam/transforms/util_test.py [hunk 1]
```diff
@@ -551,9 +551,6 @@ def test_reshuffle_preserves_timestamps(self):
               {
                   'name': 'bar', 'timestamp': 33
               },
```
Comment (angoenka) [inline sdks/python/apache_beam/transforms/util_test.py]: Can we un-sickbay this test after this fix?

**Comment by angoenka** [comment]
Retest this please

**Comment by bumblebee-coming** [comment]
R: @angoenka

File: sdks/python/apache_beam/transforms/util_test.py (hunk 1 above)
Comment (bumblebee-coming) [reply sdks/python/apache_beam/transforms/util_test.py]: I'm afraid not.
It seems b/146457921 and BEAM-9003 are two different issues.
b/146457921 is for Unified Worker and BEAM-9003 is for Dataflow.

**Comment by ananvay** [comment]
R: @robertwb

File: sdks/python/apache_beam/transforms/util_test.py (hunk 1 above)
Comment (bumblebee-coming) [reply sdks/python/apache_beam/transforms/util_test.py]: I'm not a member, I can't run the Jenkins VR test or see the results, so I can't say much about  BEAM-9003.

**Comment by HuangLED** [comment]
Run Python PreCommit

File: sdks/python/apache_beam/transforms/util_test.py (hunk 1 above)
Comment (bumblebee-coming) [reply sdks/python/apache_beam/transforms/util_test.py]: Valentyn helped reproduce BEAM-9003 bug on Dataflow. The pipeline runs forever (timeout) on Dataflow streaming. So maybe BEAM-9003 is caused by a different reason.

We'll concentrate on b/146457921 here.

Comment (bumblebee-coming) [reply sdks/python/apache_beam/transforms/util_test.py]: I'll try to run the test on VR to see if it passes.

**Comment by robertwb** [comment]
This looks good to me.

**Comment by angoenka** [comment]
LGTM
You can un-sickbay the tests separately if needed.

**Comment by bumblebee-coming** [comment]
Retest this please

**Comment by bumblebee-coming** [comment]
Retest this please

**Comment by bumblebee-coming** [comment]
This PR fixed BEAM-9003, so I unsickbayed this test.

**Comment by tvalentyn** [comment]
Run Python Dataflow ValidatesRunner

**Comment by tvalentyn** [comment]
Run Python Dataflow ValidatesRunner

**Comment by tvalentyn** [comment]
Run Python Dataflow ValidatesRunner

**Comment by bumblebee-coming** [comment]
Retest this please

**Comment by bumblebee-coming** [comment]
Run Python PreCommit

**Comment by tvalentyn** [comment]
Run Python PreCommit

**Comment by tvalentyn** [comment]
Run Python PreCommit

**Comment by tvalentyn** [comment]
Run Python PreCommit

**Comment by tvalentyn** [comment]
Run Python PreCommit

**Comment by tvalentyn** [comment]
Run PythonLint PreCommit

**Comment by tvalentyn** [comment]
Run PythonLint PreCommit

**Comment by angoenka** [comment]
Will wait for the tests to pass and merge

**Comment by angoenka** [comment]
Run PythonLint PreCommit

**Pull request merged**